src/util/webauthn.js

```diff
@@ -52,9 +52,17 @@ function getAssertion(challenge, allowList) {
   return adaptToOkta(promise);
 }
 
+function processWebAuthnResponse(inputFunction) {
+  if (typeof inputFunction === 'undefined' || inputFunction() === undefined || inputFunction() === null) {
```

Comment:
A couple of Internet posts suggest this as a check to see if a value represents a function.  It probably does not matter, but FYI.

if (typeof test === 'function') {

https://www.tutorialspoint.com/How-to-check-if-a-JavaScript-function-is-defined#:~:text=Using%20the%20typeof%20Operator,whether%20the%20function%20is%20defined.
https://stackoverflow.com/questions/1042138/how-to-check-if-function-exists-in-javascript

Reply:
We are calling the `inputFunction` multiple times here. Instead can we call it only once and check IF condition based on the response?